File: Section1 - Basics/Patterns/pattern13.js
function printTriangle(n) {
	let str = "";
	let k = 1;
	for (let i = 1; i <= n; i++) {
		for (let j = 1; j <= i; j++) {
			str = str + k + " ";
			k++;
		}
		str += "\n";
	}
	console.log(str);
}

printTriangle(5);

File: Section1 - Basics/Patterns/pattern13.java
void printTriangle(int n) {
    int k=1;
    for(int i=1; i<=n; i++) {
        for(int j=1; j<=i; j++) {
            System.out.print(k + " ");
            k++;
        }
        System.out.print("\n");
    }
}